pinc/misc.inc

```diff
@@ -251,11 +256,13 @@ function get_enumerated_param($arr, $key, $default, $choices, $allownull = false
  * @param int|null $max
  * @param bool $allownull
  *
+ * @return string|null
```

Comment:
Should be `int|null` (to match with the `?int` below)

Reply:
Good catch.